pkg/storage/couchdb/couchdbstore.go

```diff
@@ -312,6 +314,26 @@ func (c *CouchDBStore) Get(k string) ([]byte, error) {
 	return value, nil
 }
 
+// GetBulk fetches the values associated with the given keys. This method works in an all-or-nothing manner.
+// It returns an error if any of the keys don't exist. If even one key is missing, then no values are returned.
+// The end result is equivalent to calling Get(k,v) on each key-value pair individually in a loop,
+// but should be faster since this method minimizes REST calls as long as the values were stored as JSON instead of
+// attachments.
+// If values are stored as attachments, then there are still optimizations that could be done - see TODO #124.
+func (c *CouchDBStore) GetBulk(keys ...string) ([][]byte, error) {
```

Comment:
The reason for the all-or-nothing semantics: It's what I need for my EDV optimization use-case, and it simplifies things for me from the EDV-side of things. We can later add a GetBulk method that can return partial results if it's something we want.